### PR TITLE
Update authoritative chat forum (2/2)

### PR DESCRIPTION
**Pull request by mplscorwin**
Packetframe's chat forum is now on Libera.Chat

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ## PacketFrame
 
-The PacketFrame CDN is an open source (AGPL-3.0) anycast CDN platform, currently in beta. If you're interested in an account, send an email to the address listed on the front page of https://packetframe.com or reach out in #packetframe on freenode.
+The PacketFrame CDN is an open source (AGPL-3.0) anycast CDN platform, currently in beta. If you're interested in an account, send an email to the address listed on the front page of https://packetframe.com or reach out in #packetframe on the [Libera.Chat](https://freenode.net/kb/answer/chat) IRC network.
```
Comment (natesales) [inline README.md]: Let's update the link to libera.chat here

**Review by natesales — CHANGES_REQUESTED** [review]
Thanks for the PR! Just one minor thing in the review.

File: README.md (hunk 1 above)
Comment (mplscorwin) [reply README.md]: Done, I think 👯

**Review by natesales — APPROVED** [review]

**Comment by natesales** [comment]
Thank you!

**Pull request merged**